wg-deployment/charter.md

```diff
@@ -0,0 +1,35 @@
+# WG Deployment Charter
+
+This charter adheres to the conventions, roles and organization management outlined in [wg-governance].
+
+## Scope
+
+Manage manifests for deploying and operating Kubeflow.
+Develop and maintain tools like kfctl (CLI + operator) for using the manifests to deploy Kubeflow.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- kfctl
+- kfdef for each platform
+
+#### Cross-cutting and Externally Facing Processes
+
+- Cutting releases on both the manifests and kfctl repos
+- Qualifying a Kubeflow release for each platform
+- Co-ordinating with application OWNERs before a release to qualify application releases
+
+### Out of scope
+
+- Maintaining or bug fixes to the individual applications themselves.
```

Comment:
Should we explicitly call out that deciding what is and is not a conformant Kubeflow distribution is not in scope?

Reply:
I'm not so sure about this. Can we revisit this?

Reply:
Can you explain? Why would conformance be in scope given your charter and how is it relevant to maintaining kfctl and manifests?